### ISSUE TITLE
Feature req: Fractions in climate page

### ISSUE THREAD

**Issue by Floppe** [Blackymas/NSPanel_HA_Blueprint#1578]
Minor UI tweak. If climate temperature steps does not include fractions, then perhaps do not show fractions in UI either?

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/238729/4df81227-c4e3-4a5f-9132-4672a65506b0)

43.0 -> 43
60.0 -> 60

Here actual heating point could have fractions, so it could be 57.1. But min and max do not have fractions for my climate control.

**Comment by edwardtfn** [comment]
Could you please share the attributes of this entity?
I believe `target_temp_step` could be used in this case, as if it is anninteger and the `temp_target_low`/`high` are both integers, decimals shouldn't be shown.

**Comment by Floppe** [comment]
```
hvac_modes: off, auto
min_temp: 35
max_temp: 60
target_temp_step: 1
current_temperature: 54.9
target_temp_high: 60
target_temp_low: 45
hvac_action: idle
friendly_name: Tap water
supported_features: 2
```

**Comment by edwardtfn** [comment]
Ok. In fact, `min_temp` and `max_temp` should be used.

**Comment by edwardtfn** [comment]
I believe this is being fixed on v4.3 (currently in beta).
I will close this for now, but could you please reopen it if the issue persists after the update?
